code/game/gamemodes/objective.dm

```diff
@@ -224,6 +232,12 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 	if(target && owner)
 		SSticker.mode.protect_target_protectors += owner
 
+/datum/objective/protect/vip/set_explaination_text()
+	if(target && target.current)
+		explanation_text = "Protect [target.current.real_name], the [target.assigned_role]. They are believed to be hunted by agents of other companies."
+	else
+		..()
```

Comment:
Any reason why some of the parent calls for `set_explanation_text()` are `return ..()`, and some are just plain `..()`?

Reply:
No real reason other than me going on auto pilot it seems :P
Will be fixed in the next commit!